packages/components/src/font-size-picker/README.md

```diff
@@ -92,6 +92,8 @@ Size of the control.
 
 Available units for custom font size selection.
 
+**Note**: For the `units` property to work, the current font size value must be specified as strings with units (e.g., `'12px'` instead of `12`). When the font size is provided as a number, the component operates in "unitless mode" where the `units` property has no effect.
```

Comment:
What do you think about moving this to [the `value` prop section](https://github.com/WordPress/gutenberg/blob/7a4b5aa5d8a63d97b084e1339e6ce544dfc944bb/packages/components/src/font-size-picker/README.md#value-number--string)? Because unitless mode is based on the `value` prop value, not the `units` prop value.

Reply:
Let's also make sure to add these same text to the JSDocs in the `units` prop type